### ISSUE TITLE
Fluent interface in MapOptions

### ISSUE THREAD

**Issue by povilasb** [branflake2267/GWT-Maps-V3-Api#208]
It would be nice if MapOptions implemented the [Fluent Interface](http://en.wikipedia.org/wiki/Fluent_interface) pattern.

e.g.

``` java
MapOptions options = MapOptions.newInstance().setCenter(...).setZoom(...);
```

**Comment by twistedpair** [comment]
@povilasb 

Good point. This could be done without breaking changes, I believe, since presently the methods are `void`, so no one is doing anything with their return values.

I'm strapped for time at the moment, but if you've got the time to update the class, mostly just updating the method signatures and adding `return this;`, we'd be grateful and happy to merge it.

**Comment by branflake2267** [comment]
A Builder (pattern) sub type could be added as a subclass.

**Comment by twistedpair** [comment]
@branflake2267 good point and perhaps a more self contained solution.
